### PR TITLE
document core is EOL before 3.8.8 (e.g we will not provide core release for bugs nor security issues).

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

### CODE REVIEW

**Comment by michael-o** [comment]
did I miss the discussion. I thought we agreed to move to 3.5.4 as soon as all components are on Java 8 and Maven 3.2.5?

**Comment by olamy** [comment]
> Did I miss the discussion? I thought we agreed to move to 3.5.4 as soon as all components are on Java 8 and Maven 3.2.5.

Will you really release Maven Core 3.5.5 or 3.6.4?
I'm not talking about plugins or components etc. but only about Maven core distribution.

**Comment by michael-o** [comment]
> > Did I miss the discussion? I thought we agreed to move to 3.5.4 as soon as all components are on Java 8 and Maven 3.2.5.
> 
> Will you really release Maven Core 3.5.5 or 3.6.4? are we really maintaining core branches `3.2.x` `3.3.x` `3.5.x` `3.6.x` I'm not talking about plugins or components etc. but only about Maven core distribution.

No, we are not, but the minimum API we support. You don't have it the components won't run, namely plugins.

**Comment by olamy** [comment]
> > > Did I miss the discussion? I thought we agreed to move to 3.5.4 as soon as all components are on Java 8 and Maven 3.2.5.
> > 
> > 
> > Will you really release Maven Core 3.5.5 or 3.6.4? are we really maintaining core branches `3.2.x` `3.3.x` `3.5.x` `3.6.x` I'm not talking about plugins or components etc. but only about Maven core distribution.
> 
> No, we are not, but the minimum API we support. You don't have it the components won't run, namely plugins.

so this page is about core and what version of core we support (e.g maintaining and eventually releasing) that is a bit different.
As a user if looking at this page I can read it as 3.2.5 is supported and eventually 3.2.6 could be released.

**Comment by michael-o** [comment]
> > > > Did I miss the discussion? I thought we agreed to move to 3.5.4 as soon as all components are on Java 8 and Maven 3.2.5.
> > > 
> > > 
> > > Will you really release Maven Core 3.5.5 or 3.6.4? are we really maintaining core branches `3.2.x` `3.3.x` `3.5.x` `3.6.x` I'm not talking about plugins or components etc. but only about Maven core distribution.
> > 
> > 
> > No, we are not, but the minimum API we support. You don't have it the components won't run, namely plugins.
> 
> so this page is about core and what version of core we support (e.g maintaining and eventually releasing) that is a bit different. As a user if looking at this page I can read it as 3.2.5 is supported and eventually 3.2.6 could be released.

Correct, the information stated is misleading needs to be cleaned up/separation.

File: content/markdown/docs/history.md.vm [hunk 1]
```diff
@@ -61,6 +61,13 @@ $b_</td>
 
 ## Maven Releases History
 
+Apache Maven supports the last version of the last 2 series of GA release.
```
Comment (michael-o) [inline content/markdown/docs/history.md.vm:64]: 2 => two

Comment (michael-o) [inline content/markdown/docs/history.md.vm:64]: GA releases

**Review by michael-o — CHANGES_REQUESTED** [review]

File: content/markdown/docs/history.md.vm (hunk 1 above)
Comment (olamy) [reply content/markdown/docs/history.md.vm]: fixed c7f69a1d

**Review by michael-o — APPROVED** [review]
This is fine for me, let's see what others think.

**Review by slawekjaranowski — APPROVED** [review]
`supports` -> `maintains` what do you think?

It is true for me - so we will be more clear for users

**Comment by cstamas** [comment]
Agreed with @slawekjaranowski : we do not "support", we "maintain". Supporting something sounds to me like some contractual obligation, while we do not offer and "support contract" at all...

File: content/markdown/docs/history.md.vm [hunk 2]
```diff
@@ -61,6 +61,13 @@ $b_</td>
 
 ## Maven Releases History
 
+Apache Maven supports the last version of the last two series of GA releases.
```
Comment (cstamas) [inline content/markdown/docs/history.md.vm:64]: supports -> maintains

File: content/markdown/docs/history.md.vm [hunk 3]
```diff
@@ -61,6 +61,13 @@ $b_</td>
...
+If last release is 3.9.1, Apache Maven project will support core versions:
+- 3.9.1
+- 3.8.7
+
+Currently, plugins support Maven API compatibility down to 3.2.5.
```
Comment (cstamas) [inline content/markdown/docs/history.md.vm:69]: support -> provides

File: content/markdown/docs/history.md.vm (hunk 2 above)
Comment (michael-o) [inline content/markdown/docs/history.md.vm:64]: The Apache Maven team...

File: content/markdown/docs/history.md.vm [hunk 4]
```diff
@@ -61,6 +61,13 @@ $b_</td>
 
 ## Maven Releases History
 
+Apache Maven supports the last version of the last two series of GA releases.
+If last release is 3.9.1, Apache Maven project will support core versions:
```
Comment (michael-o) [inline content/markdown/docs/history.md.vm:65]: team

Comment (michael-o) [inline content/markdown/docs/history.md.vm:65]: maintain

**Review by michael-o — CHANGES_REQUESTED** [review]

**Comment by olamy** [comment]
thanks for review. changes applied

**Review by cstamas — APPROVED** [review]

File: content/markdown/docs/history.md.vm [hunk 5]
```diff
@@ -61,6 +61,13 @@ $b_</td>
 
 ## Maven Releases History
 
+The Apache Maven team maintains the last version of the last two series of GA releases.
+If last release is 3.9.1, the Apache Maven team project will maintains core versions:
```
Comment (michael-o) [inline content/markdown/docs/history.md.vm:65]: will maintain

**Review by michael-o — CHANGES_REQUESTED** [review]
Typo and then fine

**Review by michael-o — APPROVED** [review]

**Pull request merged**